meta/documents/changelog_de.md

```diff
@@ -1,5 +1,14 @@
 # Release Notes für SocialMedia
 
+## v1.1.0 (2018-03-09)
+
+### Hinzugefügt
+- Ein weiterer Container für die Anzeiger Buttons direkt am Artikel
```

Comment:
"Anzeiger Buttons" ist grammatikalisch nicht ganz sauber. Ich schlage "Anzeige-Buttons" vor.

Reply:
Ich schlage ebenfalls vor den Eintrag mit "wurde hinzugefügt" zu beschließen, das hast du in den drei letzten Einträgen ebenfalls gemacht.